theme/static/css/main.css

```diff
@@ -192,6 +192,20 @@ td {
   vertical-align: top;
 }
 
+td {
+  text-align: center;
+}
+
+td > .first {
+  text-align: center;
+}
+
```

Comment:
IIRC the `docutils` class means this element is generated by Pelican. Therefore, it's better to select the table with the `docutils` class to prevent this style from applying to our custom table

Reply:
OK, I fixed it in a7e2c0d